webapp/app.py

```diff
@@ -109,6 +77,116 @@ def _group_by_department(vacancies):
     return vacancies_by_department
 
 
+def _get_sorted_departments():
```

Comment:
This is just moving the code up, so that all `_` helper functions are in one place to be referenced.